weekly-updates/weekly-update.2015-08-14.md

```diff
@@ -0,0 +1,38 @@
+### io.js and Node.js News — August 14th
+The move of nodejs/io.js to nodejs/node, the Collaborator Summit, and more.
+
+### nodejs/io.js has moved to nodejs/node!
+[Rod Vagg](http://twitter.com/rvagg) made a [proposal](https://github.com/nodejs/node/issues/2327) regarding the convergence work between io.js and Node.js earlier this week, and one of them is about renaming the nodejs/io.js repository to [nodejs/node](https://github.com/nodejs/node). A decision has been made, and [nodejs/node](https://github.com/nodejs/node) is our new home!
+
+With the new repository name, io.js v3.x will be our last major version of io.js as the next (v4.x) will be Node.js v4.x.
+
+Other points in the proposal are:
+
+* The new repository to be the host of all future releases including io.js v3 and Node.js 0.10 and 0.12. The suggestion is that they will be in separate branches named v0.10.x and v0.12.x alongside with the existing v3.x branch.
+* Leaving [joyent/node](https://github.com/joyent/node) in place for now, encouraging all new issues and pull requests in it to be filed in nodejs/node instead. There is also a possibility to move joyent/node to nodejs/node-legacy or a similarly named repository.
+* Documenting a new, simpler version of CONTRIBUTING.md for casual contributions, as contributions need to be documented according to the new branch structure nodejs/node will have. With the new branches hosting the different versions, contributors will need to be aware which branch they need to file a pull request to and which version are their issues related to.
+
+Jump on the conversation on the [nodejs/node issue page](https://github.com/nodejs/node/issues/2327).
+
+### Community Updates
+
+* We *still* need a V8 maintainer for our LTS build! Head on over [to GitHub](https://github.com/nodejs/LTS/issues/28) to see if the requirements match your capabilities.
+* As mentioned up top, io.js v4.x will be Node.js v4.x. Or as [Mikeal Rogers](http://twitter.com/mikeal) said, [4.0 is the new 1.0](https://medium.com/node-js-javascript/4-0-is-the-new-1-0-386597a3436d).
+* The Node.js Collaborator Summit was hosted on 6th – 7th August in San Fransisco. You can see the notes in the [nodejs/summit](https://github.com/nodejs/summit) repository, [the GitHub issue](https://github.com/nodejs/summit/issues/11), or on [the Google Docs folder](https://drive.google.com/folderview?id=0B4tYrG9tjAW0fk9VOG52R2VQMmZHNWdpYnpqWU5MYUlfamNycktmeEl3UkNhdk1PS1Rid3c&usp=sharing).
+* [nodejs/collaboration](https://github.com/nodejs/collaboration) was made along with [the team](https://github.com/orgs/nodejs/teams/collaboration) to help with communication between Working Groups, as well as getting better ways to have evangelism, website, documentation and the 30+ i18n groups work together.
+* The new [Intl WG](https://github.com/nodejs/Intl) had [a meeting](https://github.com/nodejs/Intl/issues/8/?utm_source=io.js+and+Node.js+News&utm_medium=article) on August 11th.
```

Comment:
> https://github.com/nodejs/Intl/issues/8/?utm_source=io.js+and+Node.js+News&utm_medium=article

The url has querystring for tracking source, we would be better to remove those queries.